R/linter_tag_docs.R

```diff
@@ -99,3 +99,11 @@ NULL
 #' @evalRd rd_linters("executing")
 #' @seealso [linters] for a complete list of linters available in lintr.
 NULL
+
+#' Testthat linters
+#' @name testthat_linters
+#' @description
+#' Linters encouraging best practices within testthat suites.
```

Comment:
This could use some external references to testthat docs, preferably about best practices.

Reply:
I just linked to the testthat site, I don't know of any "best practices" docs they produce. Even if so I worry about introducing link rot.

Reply:
SGTM. Maybe also reference the R Packages section?

https://r-pkgs.org/testing-basics.html